pipeline/acquisition.py

```diff
@@ -160,21 +160,22 @@ def make(self,key):
         key['current_injection'] = np.array(nwb['acquisition']['timeseries']['current_injection']['data'])
         key['current_injection_time_stamps'] = np.array(nwb['acquisition']['timeseries']['current_injection']['timestamps']),
         self.CurrentInjection.insert1(key, ignore_extra_fields=True)
+        nwb.close()
 
     
 @schema
-class Extracellular(dj.Manual):
+class ProbeInsertion(dj.Manual):
     definition = """ # Information relating to the extracelluar recording (e.g. location, probe)
```

Comment:
In the table comment describe the entity represented by each row.  "Information" is too vague.